### PR TITLE
Update table/prometheus-cloudwatch-exporte

### PR DESCRIPTION
**Pull request by AdamDang**
Line 44: tables lists->table lists

### CODE REVIEW

**Comment by AdamDang** [comment]
ests failed on CircleCI, Close this PR first.

**Comment by AdamDang** [comment]
Add home key for prometheus-cloudwatch-exporter

File: stable/prometheus-cloudwatch-exporter/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,8 @@ apiVersion: v1
 appVersion: "0.1.0"
 description: A Helm chart for prometheus cloudwatch-exporter
 name: prometheus-cloudwatch-exporter
-version: 0.1.0
+version: 0.1.1
+home: https://prometheus.io
```
Comment (gianrubio) [inline stable/prometheus-cloudwatch-exporter/Chart.yaml]: should be https://github.com/prometheus/cloudwatch_exporter

**Comment by AdamDang** [comment]
@gianrubio    modify the home key "https://github.com/prometheus/cloudwatch_exporter"

**Pull request merged**